h2o-genmodel/src/main/java/hex/genmodel/AbstractMojoWriter.java

```diff
@@ -180,6 +180,9 @@ private void addCommonModelInfo() throws IOException {
     writekv("n_classes", model.nclasses());
     writekv("n_columns", model.columnNames().length);
     writekv("n_domains", n_categoricals);
+    if (model.offsetColumn() != null) {
```

Comment:
Having it for all makes more sense. Great.

Reply:
thank you